### PR TITLE
feat: add createBucket & getBucket method

### LINKED ISSUE #31: Add support for creating a new bucket
It would be nice to support the creation of a new bucket.

### PR DESCRIPTION
**Pull request by c4spar**
closes #31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,8 +11,19 @@ Amazon S3 for Deno
...
 
+// Create a new bucket.
+const s3 = new S3({
+  accessKeyID: Deno.env.get("AWS_ACCESS_KEY_ID")!,
+  secretKey: Deno.env.get("AWS_SECRET_ACCESS_KEY")!,
+  region: "us-east-1",
+  endpointURL: Deno.env.get("S3_ENDPOINT_URL"),
+});
+
+const myBucket = s3.createBucket("my-bucket", { acl: "private" });
```
Comment (lucacasonato) [inline README.md]:
```suggestion
const myBucket = await s3.createBucket("my-bucket", { acl: "private" });
```

**Review by lucacasonato — COMMENTED** [review]
I think there should be a way to get from `S3` to an existing `S3Bucket` with all the details  access keys etc set.

**Comment by c4spar** [comment]
I agree, I also suggested in one of the other PR's to add a `getBucket` method. Will do 👍

**Comment by c4spar** [comment]
I think I'm done 🙂

**Review by lucacasonato — APPROVED** [review]
LGTM! Thank you :-)

**Pull request merged**